File: Code08/Basics/bubble.java
class FindPivot{
	static void bubblesort(int a[],int x,int n){
		for(int i = 0;i<=n;i++){
			for(int j=i+1;j<=n;j++){
				if(a[i]>a[j]){
					int temp = a[i];
					a[i] = a[j];
					a[j] = temp;
				}
			}
		}
		for(int i=0;i<=n;i++){
			System.out.println(a[i]);
		}
	}
	
	public static void main(String[] args){
		int[] a = new int[]{7,8,9,10,1,2,3,4,5,6};
		bubblesort(a,0,9);
	
	}
 
}

File: Golang/2020Go/bubble.go
package main

import (
	"fmt"
)

func bubbleSort(array []int, size int) {
	for i := 0; i < size; i++ {
		for j := 0; j < size; j++ {
			if array[i] < array[j] {
				var temp int
				temp = array[i]
				array[i] = array[j]
				array[j] = temp
			}
		}
	}
}

func main() {
	fmt.Println("Enter the size of array")
	var size int
	fmt.Scan(&size)

	array := make([]int, size, 100)

	for i := 0; i < size; i++ {
		fmt.Println("Enter element : ")
		fmt.Scan(&array[i])
	}

	bubbleSort(array, size)

	fmt.Println("Sorted Array : ")

	for i := 0; i < size; i++ {
		fmt.Println(array[i])
	}

}
